profile-widget/divecartesianaxis.cpp

```diff
@@ -106,8 +106,22 @@ int DiveCartesianAxis::getMinLabelDistance(const DiveCartesianAxis &timeAxis) co
 	return int(ceil(interval));
 }
 
-static double sensibleInterval(double inc, int decimals)
+static double sensibleInterval(double inc, int decimals, bool is_time_axis)
 {
+	if (is_time_axis && inc < 60.0) {
+		// for time axes and less than one hour increments, round to
+		// 1, 2, 3, 4, 5, 6  or 12 parts of an hour or a minute.
```

Comment:
I'm intrigued by these choices... I would have expected 1,2,3,4,5,6,10,15,20 or something like that
oh, wait... those are how many parts we have? so this is actually 60, 30, 20, 15, 12, 10, 5?
If that's the case, then the explanation could be improved :)

Reply:
Added the corresponding reciprocals.